src/lib/seed-db.js

```diff
@@ -29,13 +30,13 @@ function * setupAdminAccount (adminParams) {
     adminAccount.fingerprint = adminParams.fingerprint
     yield upsertAccount(adminAccount)
   } else {
-    yield upsertAccount({
+    yield upsertAccount(converters.convertToInternalAccount({
```

Comment:
I changed this line to use [insertAccounts()](https://github.com/interledgerjs/five-bells-ledger/blob/master/src/models/accounts.js#L98).

I think it is more robust to pass in an external account to take advantage of all the input validation that is done during the conversion from external to internal account. In the future, [small changes to the admin account setup](https://github.com/interledgerjs/five-bells-ledger/commit/e994f2f7c7713bc2ce8fb3acaa20348c577e93e2) are less likely to result in failing database queries.